app/questionnaire/placeholder_parser.py

```diff
@@ -40,9 +40,11 @@ def __call__(self, placeholder_list: Sequence[Mapping]) -> Mapping:
 
     def _lookup_answer(
         self, answer_id: str, list_item_id: str = None
-    ) -> Union[Markup, None]:
+    ) -> Union[Markup, Sequence[Markup], None]:
         answer = self._answer_store.get_answer(answer_id, list_item_id)
         if answer:
+            if isinstance(answer.value, list):
```

Comment:
The use of isinstance is generally seen as a bad code smell or data choice. For example, in this example, if the value was always an iterable, the isinstance could be omitted. Instead we now have multiple return types for the placeholder_parser as well. This isn't a blocking issue for this PR, but it's worth mentioning as it is tech debt that could be refined.

Reply:
I'd considered the alternatives, but the fact is the value can be multiple types.

Refactoring the answerstore in this PR certainly seems overkill given that we frequently use isinstance elsewhere.

Reply:
Yes, definitely overkill in this instance - are we tracking this kind of tech debt anywhere @ajmaddaford ?

Reply:
Tech debt is tracked by raising a card with a 'Tech Debt' label. I don't think we have one for this so please add one. Ensure you add an explanation of why it's an issue and if possible a proposed fix.